Source/AwsCommonRuntimeKit/crt/ByteCursor.swift

```diff
@@ -3,45 +3,13 @@
 import AwsCCommon
 import Foundation
 
-public protocol ByteCursor {
-    var rawValue: aws_byte_cursor { get set }
-}
-
-extension aws_byte_cursor: ByteCursor {
-    public var rawValue: aws_byte_cursor {
-        get { return self }
-        set(value) {
-            self = value
-        }
-    }
-}
-
 extension aws_byte_cursor {
-    public func toString() -> String? {
+    func toString() -> String? {
         if self.len == 0 { return nil }
-        return String(bytesNoCopy: self.ptr, length: self.len, encoding: .utf8, freeWhenDone: false)
+        return String(bytes: UnsafeBufferPointer(start: self.ptr, count: self.len), encoding: .utf8)
```

Comment:
BytesNoCopy is no longer supported. https://developer.apple.com/documentation/swift/string/init(bytesnocopy:length:encoding:freewhendone:)